### PR TITLE
Fix a11y logo issue

### PR DESCRIPTION
**Pull request by Molochem**
As requested in https://github.com/plone/volto/issues/5720 , i've implemented the changes

### CODE REVIEW

**Review by stevepiercy — CHANGES_REQUESTED** [review]
There was still disagreement on how to proceed. I think this PR should be closed until there is agreement.

Aside from that "Front Page" is seldom used for websites. It's more for newspapers, when there was such a thing 😜 .

"Home" is much more common usage and already exists in the language files, so let's reuse that.

We definitely do not want to include "Back to ". How do you know that someone came from that location and not directly to that page?

**Comment by tisto** [comment]
@stevepiercy let's keep the discussion in the issue: https://github.com/plone/volto/issues/5720#issuecomment-1956209485

File: README.md
Comment (stevepiercy) [inline README.md:1]: We can ignore the failure for these two links. Let's give the owner a chance to fix the links in a separate PR. If they don't respond within a reasonable time frame, then we can remove them in separate PR. We can merge your PR without them fixed.

File: packages/volto/news/5776.breaking [hunk 1]
```diff
@@ -0,0 +1 @@
+implemented requested changes, added i18n , changed type of news fragment @Molochem
```
Comment (stevepiercy) [inline packages/volto/news/5776.breaking:1]:
```suggestion
Improved accessibility of logo component. @Molochem
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
This should also be reviewed by the commenters on the original issue. This looks much better. Thank you!

File: README.md
Comment (stevepiercy) [reply README.md:1]: We removed the links, and I resolved a merge conflict.

File: packages/volto/src/components/theme/Logo/__snapshots__/Logo.Multilingual.test.jsx.snap [hunk 2]
```diff
@@ -2,52 +2,50 @@
...
   className={null}
   href="/en"
   onClick={[Function]}
   target={null}
-  title="English"
+  title={null}
 >
   <img
-    alt="English"
+    alt="Logo of undefined"
```
Comment (sneridagh) [inline packages/volto/src/components/theme/Logo/__snapshots__/Logo.Multilingual.test.jsx.snap:46]: Can we implement the mock so the test does not throw "undefined"?

**Review by sneridagh — DISMISSED** [review]

**Review by JeffersonBledsoe — APPROVED** [review]

File: packages/volto/src/components/theme/Logo/__snapshots__/Logo.Multilingual.test.jsx.snap (hunk 2 above)
Comment (Molochem) [reply packages/volto/src/components/theme/Logo/__snapshots__/Logo.Multilingual.test.jsx.snap]: Done

File: packages/volto/src/components/theme/Logo/Logo.jsx [hunk 3]
```diff
@@ -34,26 +46,20 @@ const Logo = () => {
...
 
   return (
-    <ConditionalLink
-      href={navRootPath}
-      title={navroot?.navroot?.title}
-      // In case that the content returns 404, there is no information about the portal
-      // then render the link anyways to get out of the Unauthorized page
-      condition={!navroot || currentURLIsNavRoot}
-    >
+    <a href={navRootPath} aria-label={intl.formatMessage(messages.home)}>
```
Comment (sneridagh) [inline packages/volto/src/components/theme/Logo/Logo.jsx:51]: @Molochem I just realised of this. Is there any rationale behind it is not a react router link but a plain link?

Comment (sneridagh) [reply packages/volto/src/components/theme/Logo/Logo.jsx:51]: https://github.com/plone/volto/pull/5981

Comment (Molochem) [reply packages/volto/src/components/theme/Logo/Logo.jsx:51]: Nope , just an oversight by me. Thx for fixing.

**Pull request merged**